### PR TITLE
[#292] Navigate to create user when no user exists (fixes #292)

### LINKED ISSUE #292: Make user create option when no admin
When application is newly setup and no admin at all take to create user not to login

### PR DESCRIPTION
**Pull request by yaoxing1990**

### CODE REVIEW

**Comment by nirojdyola** [comment]
@paulbert  and @lmmrssa Do we need this feature ? I think it has been already implemented on login section.

File: src/app/login/login.component.ts [hunk 1]
```diff
@@ -25,6 +25,14 @@ export class LoginComponent {
   model = { name: '', password: '', repeatPassword: '' };
   message = '';
 
+  ngOnInit() {
```
Comment (Rupesh87) [inline src/app/login/login.component.ts]: you must also check if admin is exist or not

Comment (yaoxing1990) [reply src/app/login/login.component.ts]: @Rupesh87 I think "no user" exists means no admin

Comment (paulbert) [reply src/app/login/login.component.ts]: @yaoxing1990 No documents in the `_users` database does not mean no admin.  CouchDB stores admins in the main configuration database.

**Comment by paulbert** [comment]
Closing this because it duplicates #296

**Pull request closed without merging**